cli/internal/cmd/configfetchmeasurements_test.go

```diff
@@ -78,17 +77,17 @@ func TestParseFetchMeasurementsFlags(t *testing.T) {
 			require := require.New(t)
 
 			cmd := newConfigFetchMeasurementsCmd()
-			cmd.Flags().String("config", constants.ConfigFilename, "") // register persistent flag manually
-			cmd.Flags().Bool("force", false, "")                       // register persistent flag manually
+			cmd.Flags().String("workspace", "", "") // register persistent flag manually
```

Comment:
nit: nothing to be changed here but it smells to me that we need to do that. Just an idea. Can we test the CLI from a user perspective?
Like here: https://gianarb.it/blog/golang-mockmania-cli-command-with-cobra
In our case we would wrap the cobra command to be able to mock the external dependencies of the run function.

Reply:
Possible, and maybe something we want to do in the future, but I think that would be way out of scope for this PR

Reply:
yeah not for this PR, but will add a backlog item to discuss.